<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrowWindow</class>
 <widget class="QMainWindow" name="CrowWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/common/crow_icon/crow.ico</normaloff>:/common/crow_icon/crow.ico</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="rightLayout">
      <item>
       <widget class="sore::CrowPlayerWrapper" name="videoPlayer">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="sore::HideableDockWidget" name="subtitleDockWidget">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Subtitle Viewer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="subtitleDockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="sore::SubtitleListView" name="subtitleList">
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="sore::HideableDockWidget" name="episodesDockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Episode List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="episodesDockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListView" name="episodeList">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="sore::EpisodeManager" name="episodeManager" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuAudio">
    <property name="title">
     <string>Audio</string>
    </property>
    <widget class="QMenu" name="menuAudioTrack">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Audio Track...</string>
     </property>
     <addaction name="actionPlaceholderAudio"/>
    </widget>
    <widget class="QMenu" name="menuAudioDevice">
     <property name="title">
      <string>Audio Device...</string>
     </property>
     <addaction name="actionDevicePlaceholder"/>
    </widget>
    <addaction name="menuAudioTrack"/>
    <addaction name="menuAudioDevice"/>
   </widget>
   <widget class="QMenu" name="menuSubtitle">
    <property name="title">
     <string>Subtitle</string>
    </property>
    <widget class="QMenu" name="menuSubtitleTrack">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Subtitle Track...</string>
     </property>
     <addaction name="actionPlaceholderSubtitle"/>
    </widget>
    <widget class="QMenu" name="menuSecondarySubtitle">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Secondary Subtitle...</string>
     </property>
     <addaction name="actionempty"/>
    </widget>
    <addaction name="menuSubtitleTrack"/>
    <addaction name="menuSecondarySubtitle"/>
    <addaction name="actionAddExternalTrack"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAudio"/>
   <addaction name="menuSubtitle"/>
  </widget>
  <action name="actionNewProject">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionPlaceholderAudio">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actionDevicePlaceholder">
   <property name="text">
    <string>empty</string>
   </property>
  </action>
  <action name="actionPlaceholderSubtitle">
   <property name="text">
    <string>Disable</string>
   </property>
  </action>
  <action name="actionAddExternalTrack">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add external track...</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionempty">
   <property name="text">
    <string>Disable</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>sore::HideableDockWidget</class>
   <extends>QDockWidget</extends>
   <header>gui/components/hideable_dock/hideable_dock.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>sore::CrowPlayerWrapper</class>
   <extends>QOpenGLWidget</extends>
   <header>gui/widgets/crow_player_wrapper/crow_player_wrapper.h</header>
  </customwidget>
  <customwidget>
   <class>sore::SubtitleListView</class>
   <extends>QListView</extends>
   <header>gui/widgets/subtitle_list_view/subtitle_list_view.h</header>
  </customwidget>
  <customwidget>
   <class>sore::EpisodeManager</class>
   <extends>QWidget</extends>
   <header>gui/widgets/episode_manager/episode_manager.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../sore_crow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>CrowWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
